nrcatalogtools/waveform.py

```diff
@@ -165,7 +165,7 @@ def load_from_h5(cls, file_path_or_open_file, metadata={}, verbosity=0):
             data[:, idx] = amp_interp(times) * np.exp(1j * phase_interp(times))
 
         w_attributes = {}
-        w_attributes["metadata"] = metadata
+        # w_attributes["metadata"] = metadata
```

Comment:
No need to comment this line out, please remove it entirely. Please also remove the other commented out # w_attribute below.

Reply:
Sure, just did.